concent_api/api-integration-force-report-computed-task-test.py

```diff
@@ -72,7 +72,7 @@ def ack_report_computed_task(task_id, provider_private_key, provider_public_key,
 def main():
     cluster_url     = parse_command_line(sys.argv)
     task_id         = random.randrange(1, 100000)
-    current_time    = int(datetime.datetime.now().timestamp())
+    current_time    = calendar.timegm(time.gmtime())
```

Comment:
Please use the helpers in these scripts too.

Reply:
:+1: I have checked, in current implementation cluster tests run from pure python doesn't parse settings but usage of timezone.now() without settings return time in UTC.